lib/bolt_ext/server.rb

```diff
@@ -10,10 +10,24 @@ class TransportAPI < Sinatra::Base
   # This disables Sinatra's error page generation
   set :show_exceptions, false
 
+  def initialize(app = nil)
+    @executor = Bolt::Executor.new(0, load_config: false)
+    super(app)
+  end
+
   get '/' do
     200
   end
 
+  get '/admin/gc' do
```

Comment:
Exposing this via the API concerns me... Could we gate this behind `RACK_ENV` for dev purposes only? I'm not quite as worried about the `/gc_stats` endpoint.

Reply:
Yeah, I can add that.